src/lib/pagination/ctrl.coffee

```diff
@@ -0,0 +1,99 @@
+EventsEmitter = require( 'events' )
+Pagination = require( './index' )
+PaginationView = require( './view' )
+
+###
+ * #PaginationCtrl
+ * Creates a pagination element and controls it.
+ *
+ * ## Events
+ *
+ * | CONSTANT.argument  | Description |
+ * |--------------------| -------------|
+ * | CHANGE | When page number is changed by user |
+ * | CHANGE.current  | New page number |
+ * | CHANGE.last | Number of last page |
+###
+class PaginationCtrl extends EventsEmitter
```

Comment:
[ PESAO MODE ON ]
Ooook... but then... are you aware that in both files that you extend from EventEmitter (this one, and GridPaginationView) you have an instance of "Pagination" stored in the @pagination property... and this already have an instance of EventEmitter?

So maybe you could avoid the extends on both files if you do:

@pagination.emit( @CHANGE, current, last )
instead of
@emit( @CHANGE, current, last )

P.D: Ya me callo.